docker/dropin_env_base/Dockerfile

```diff
@@ -3,7 +3,14 @@ FROM openjdk:11.0.16-jre-slim-bullseye
 
 ARG DATAROBOT_MLOPS_VERSION
 
-RUN apt update && apt upgrade -y && apt install -y python3 python3-pip python3-dev python3-distutils fontconfig nginx && \
+RUN apt update && apt upgrade -y && apt install -y wget build-essential libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev libffi-dev zlib1g-dev fontconfig nginx && \
+    cd /tmp/ && \
+    wget https://www.python.org/ftp/python/3.11.4/Python-3.11.4.tgz && \
+    tar -xzvf Python-3.11.4.tgz && \
+    cd Python-3.11.4/ && \
+    ./configure --enable-optimizations && \
+    make -j `nproc` && \
+    make install && \
```

Comment:
@yakov-g do you know if installing Python from source code results in a bigger image size? If so, what's the difference? (If there is a significant difference, I'd recommend using the multi-stage build:
https://docs.docker.com/build/building/multi-stage/#name-your-build-stages)

Reply:
Yes, new image is 1.1GB bigger. 
It's probably not only because of python but all the toolchain.

So how stages should help? Every stage would be just a separate image, which doesn't nee to be rebuilt?

Also, do you want me to use stages in this PR? Base image is already pushed to dockerhub.

Reply:
Eventually, after installin Py3.11 from source images became 7.3Gb vs 6.3 with py 3.9.

So I implemented py3.11 installation baed on the UBI image. it is ~6.4GB